c/embeddedjs.c

```diff
@@ -1985,16 +1985,15 @@ JSModuleDef *ejsModuleLoader(JSContext *ctx,
     size_t buf_len;
     uint8_t *buf;
     JSValue func_val;
-    char *nameToLoad = (char*)moduleName;
+    char nameToLoad[modNameLen + 3 + 1];
```

Comment:
Could you use `sizeof(asciiDotJS)` instead of `+ 3`. I know we'll waste 1 byte, but this would be clearer.

Reply:
done, thanks